modules/installation-vsphere-infrastructure.adoc

```diff
@@ -35,7 +35,7 @@ You can host the VMware vSphere infrastructure on-premise or on a link:https://c
 
 |Hypervisor
 |vSphere 7.0 Update 2 and later with virtual hardware version 15
-|This version is the minimum version that {op-system-first} supports. See the link:https://access.redhat.com/ecosystem/search/#/ecosystem/Red%20Hat%20Enterprise%20Linux?sort=sortTitle%20asc&vendors=VMware&category=Server[Red Hat Enterprise Linux 8 supported hypervisors list].
+|This version is the minimum version that {op-system-first} supports. For more information about supported hardware on the latest version of {op-system-base-full} that is compatible with {op-system}, see link:https://catalog.redhat.com/hardware/search[Hardware] on the _Red Hat Customer Portal_.
```

Comment:
I'd suggest removing italic formatting from "Red Hat Customer Portal". Although I can't find a style guide reference that says you can't, none of the references to the portal I found in the docs have italics.

Reply:
Thanks, @skopacz1 and @adellape . I struggled to find guidance also but I used the format I used to adopt in the Runtimes project space. I did see this entry in our `doc_guidelines.adoc` file if this adds more clarity to my use of italics: 

![Screenshot from 2023-07-12 09-32-00](https://github.com/openshift/openshift-docs/assets/57954076/b7667747-978c-4e2a-bcf0-9a97fdfdf72a)

Reply:
WDYT? Keep italic format or remove it?

Reply:
I think this situation is a bit weird because we're not referencing any explicit set of documentation (which I believe our style guides provide guidance for), we're talking about a site where you can access a host of things.

Personally I'd still be in favor of removing the italic formatting. Alex mentioned that if there's no explicit guidance, this Support page would be a good model to follow: [Support overview](https://docs.openshift.com/container-platform/4.13/support/index.html#:~:text=Red%20Hat%20Customer%20Portal)

Ultimately it's up to you, but I think keeping it plain format would keep it more consistent with other instances in our documentation. Either way this is likely something worth clarifying in some style or contribution guide.